easypost/requestor.py

```diff
@@ -227,7 +230,7 @@ def requests_request(
                 params=url_params,
                 headers=headers,
                 json=body,
-                timeout=timeout,
+                timeout=self.client.timeout if self.client else timeout,  # TODO: Remove fallback
```

Comment:
question: I know we previously only define one timeout, can we have a tuple to set up the connection and open timeouts? Below is the link I was reading.

https://requests.kennethreitz.org/en/latest/user/advanced/#timeouts